content/_changelogs/10.11.0.md

```diff
@@ -0,0 +1,32 @@
+## 10.11.0
+
+_Released 10/25/2022_
+
+**Features:**
+
+- Adds support for [`cy.session()`](/api/commands/session) to experimental WebKit.
+Ensures that with `experimentalSessionAndOrigin` enabled, tests are still able to run against webkit. Addresses [#24116](https://github.com/cypress-io/cypress/issues/24116), [#23832](https://github.com/cypress-io/cypress/issues/23832).
+- Improves user experience when connecting a project to the Dashboard. Addresses [#23379](https://github.com/cypress-io/cypress/issues/23379), [#23766](https://github.com/cypress-io/cypress/issues/23766), [#23767](https://github.com/cypress-io/cypress/issues/23767).
+  - Users can now connect a project to the Dashboard when logging in from the Cypress Launchpad after a testing type has been selected.
+  - Users will be prompted to record runs to the Dashboard from both the Cypress App and Cypress Launchpad if a project is connected to the Dashboard and has no recorded runs.
+
+**Bugfixes:**
+
+- [`cy.origin()`](/api/commands/origin) now supports more than 30 unique origin spec bridges per test. Fixes [#22874](https://github.com/cypress-io/cypress/issues/22874), [#23967](https://github.com/cypress-io/cypress/issues/23967).
+- Fixed an issue where `document.cookie` would not reflect the correct value in cross-origin tests. Fixes [#23531](https://github.com/cypress-io/cypress/issues/23531)
+- Fixed as issue where browser-skipped tests were incorrectly recorded to the Dashboard which resulted in the Dashboard marking the test as "new" or "modified" when it already existed. Fixes [#23517](https://github.com/cypress-io/cypress/issues/23517).
```

Comment:
Lets put the last bullet first and the dashboard bullet second. Generally we list this in the order of significance for users that skim quickly

Reply:
Moved items as suggested